tests/webview/ui/test_sitemap.py

```diff
@@ -31,3 +42,47 @@ def test_sitemap_is_segmented_by_author(webview_base_url, selenium, author_usern
     url_region = sitemap.url_regions[0]
     content = url_region.open()
     assert author_name in content.content_header.authors
+
+
+@markers.webview
+@markers.smoke
+@markers.requires_varnish_routing
+@markers.nondestructive
+@markers.parametrize(
+    "author, uuid, should_be_presence",
+    [  # case parameters for the removal of derived content
+        (
+            "cnxbio_espanol",
+            # Conceptos de Biología
+            "e7a016d3-91fc-4ba0-9e05-a33e986f3d94",
+            False,
+        ),
+        # case parameters for keeping OpenStax derived content (i.e. REX book)
+        (
+            "OpenStaxCollege",
+            # Introduction to Sociology 2e
+            "02040312-72c8-441e-a685-20e9333f3e1d",
+            True,
+        ),
+    ],
+)
+def test_derived_copies_in_sitemap(webview_base_url, author, uuid, should_be_present):
+    # As OpenStax, I want to remove all derived copies except OpenStax
+    # books from the content, so that we begin the transition away from cnx.
+    # see: https://github.com/openstax/cnx/issues/727
+
+    # GIVEN a "sitemap" URL
+    # note, this is discovered by the spider as part of /sitemap_index.xml
+    url = f"{webview_base_url}/sitemap-{author}.xml"
+
+    # WHEN a spider requests the specific author's sitemap
+    response = requests.get(url)
+    response.raise_for_status()
+    xml = lxml.etree.fromstring(response.content)
+    #: give the text for the the element named 'loc' in any namespace
+    urls = xml.xpath("//*[local-name()='loc']/text()")
+
+    # THEN non-OpenStax derived copies have been removed
+    # while OpenStax books remain
+    found_url = [url for url in urls if uuid in url]
+    assert xnor(should_be_present, found_url)
```

Comment:
@m1yag1 do you feel this is better or worse? 😬

Reply:
https://en.wikipedia.org/wiki/XNOR_gate

Reply:
hmm.. i might have to run it manually but based on the truth table.
`found_url = True`
`should_be_present = False`
`xnor(should_be_present, found_url)` == True. 

Wouldn't we want `should_be_present` False to fail if there was a found_url?

Also the opposite, if should_be_present is True and found urls is False we would get True.

I could also not be understanding exactly what's going on here. That is entirely possible as it's after 2pm when my brain goes retarded.

Reply:
I'll run it manually. i do like the approach tho. it's more readable and uses boolean logic 👍

Reply:
Actually i just noticed my mistake... haha. i completely didn't see the not in the xor function derp

Reply:
can you add the wiki link as a comment somewhere?

Reply:
can say something like #see truth table: https://en.wikipedia.org/wiki/XNOR_gate